.claude/scripts/verify.sh

```
@@ -0,0 +1,396 @@
+    local auto_fixed_count
+
+    cd "$PROJECT_ROOT" || exit 1
+
+    if [ "$MODE" = "staged" ]; then
+        compliance_args="--staged --autofix"
+    else
+        compliance_args="--all --autofix"
+    fi
+
+    compliance_output=$(uv run python .claude/scripts/claude-md-compliance.py $compliance_args 2>/dev/null)
+    compliance_exit=$?
```

Shellcheck (0.11.0)

[warning] compliance_exit appears unused. Verify use (or export if used externally).

(SC2034)